tests/test_schemas.py

```
@@ -0,0 +1,443 @@
+                "from": "sender@example.com",
+                "to": ["recipient@example.com"],
+                "subject": "Test",
+                "body": "Test body"
+            }
+        )
+
+
+def test_email_request_missing_required_fields():
+    """Test email request missing required fields"""
+    with pytest.raises(ValidationError):
+        EmailRequest(**{"caller_id": "test-caller"})
```

Ruff (0.14.10)

Unnecessary `dict` kwargs

Remove unnecessary kwargs

(PIE804)